client/src/pages/Payment.js

```diff
@@ -0,0 +1,298 @@
+    alignItems: "flex-start",
+  },
+  margin: {
+    margin: theme.spacing(4, 4),
+  },
+
+  form: {
+    width: "30vw",
+    alignSelf: "center",
+
+    borderRadius: "7px",
+    padding: "40px",
+  },
+
+  input: {
+    borderRadius: "6px",
+    marginBottom: "6px",
+    padding: "12px",
+    border: "1px solid rgba(50, 50, 93, 0.1)",
+    maxHeight: "44px",
+    fontSize: "16px",
+    width: "100%",
+    background: "white",
+    boxSizing: "border-box",
+  },
+  container: {
+    display: "flex",
+  },
+
+  left__container: {
+    display: "flex",
+    flexDirection: "column",
+    flex: "0.6",
+    height: "90vh",
+    background: "white",
+    overflowY: "hidden",
+  },
+  right__container: {
+    display: "flex",
+    flexDirection: "column",
+    flex: "0.4",
+  },
+  logo: {
+    margin: "20px",
+    padding: "20px",
+    borderBottom: "1px solid lightgrey",
+  },
+
+  order: {
+    marginTop: "50px",
+    marginLeft: "50px",
+    marginRight: "50px",
+    background: "white",
+    padding: "20px",
+
+    "& h4": {
+      textAlign: "center",
+    },
+  },
+
+  orderDetails: {
+    display: "flex",
+    "& p": {
+      marginTop: "30px",
+      textAlign: "left",
+    },
+  },
+
+  btn: {
+    borderRadius: "0",
+    height: theme.spacing(7),
+    margin: "0 50px",
+    paddingLeft: "20px",
+    paddingRight: "20px",
+    textTransform: "capitalize",
+    fontWeight: "500",
+  },
+  cardError: {
+    margin: "20px",
+    color: "rgb(105, 115, 134)",
+    fontSize: "16px",
+    lineHeight: "20px",
+    marginTop: "12px",
+    textAlign: "center",
+  },
+
+  img: {
+    height: "100px",
+    width: "100px",
+    margin: "20px 20px",
+  },
+
+  orderTotal: {
+    color: "#FF743D",
+    fontWeight: "bold",
+  },
+
+  paymentDetails: {
+    marginLeft: "12px ",
+    marginBottom: "20px ",
+  },
+
+  title: {
+    paddingLeft: "60px",
+  },
+
+  resultMessage: {
+    lineHeight: "22px",
+    fontSize: "16px",
+  },
+
+  hidden: {
+    display: "none",
+  },
+}));
+
+const cardStyle = {
+  style: {
+    base: {
+      color: "#32325d",
+      fontFamily: "Arial, sans-serif",
+      fontSmoothing: "antialiased",
+      fontSize: "16px",
+      "::placeholder": {
+        color: "#32325d",
+      },
+    },
+    invalid: {
+      color: "#fa755a",
+      iconColor: "#fa755a",
+    },
+  },
+};
+
+const Payment = ({ history }) => {
+  const classes = useStyles();
+
+  const [succeeded, setSucceeded] = useState(false);
+  const [error, setError] = useState(null);
+  const [processing, setProcessing] = useState("");
+  const [disabled, setDisabled] = useState(true);
+  const [clientSecret, setClientSecret] = useState("");
+  const stripe = useStripe();
+  const elements = useElements();
+
+  // data from context
+  const { userInfo } = useContext(UserContext);
+
+  useEffect(() => {
+    if (!userInfo) {
+      history.replace("/login");
+    }
+
+    // Create PaymentIntent as soon as the page loads
+
+    window
+      .fetch("/payment", {

```

Comment:
You don't need to call the window object here, the browser will find anything on the window object w/out it 👍 
```suggestion
    fetch("/payment", {

```